pkg/sfu/downtrack.go

```diff
@@ -1561,17 +1583,71 @@ func (d *DownTrack) sendPaddingOnMute() {
 	// let uptrack have chance to send packet before we send padding
 	time.Sleep(waitBeforeSendPaddingOnMute)
 
-	for i := 0; i < maxPaddingOnMute; i++ {
+	if d.kind == webrtc.RTPCodecTypeVideo {
+		d.sendPaddingOnMuteForVideo()
+	} else if d.mime == "audio/opus" {
+		d.sendSilentFrameOnMuteForOpus()
+	}
```

Comment:
we don't have a use case yet, but wondering if there needs to be support for `audio/red` here too for completeness?

Reply:
I think we can do that with support red in go-sdk